js/views/settingsVw.js

```diff
@@ -923,7 +923,10 @@ module.exports = Backbone.View.extend({
       }
     });
 
-    this.blockedUsersVw.remove();
+    // We haven't previously viewed the list of blocked users
+    if (typeof this.blockedUsersVw !== 'undefined') {
+      this.blockedUsersVw.remove();
+    }
 
```

Comment:
Good catch. 

This could be more concisely written as 

this.blockedUsersVw && this.blockedUsersVw.remove();

But the logic's the same. I'll merge and then make it shorter later.